### ISSUE TITLE
A question of BatchA2CLoss

### ISSUE THREAD

**Issue by yata0** [deepmind/open_spiel#962]
[loss of a2c rl_losses.py](https://github.com/deepmind/open_spiel/blob/c3f8b538afd6223d450c0f74269937e76850cf33/open_spiel/python/algorithms/losses/rl_losses.py#L196)
I think the total loss should be 
`total_loss = policy_loss - entroy_loss` 
instead of
` total_loss = tf.add(
          total_loss, entropy_loss, name="total_loss_with_entropy")`
@lanctot @elkhrt

**Comment by lanctot** [comment]
I _think_ what we have is correct, because the `compute_entropy` returns negative entropy:

https://github.com/deepmind/open_spiel/blob/c3f8b538afd6223d450c0f74269937e76850cf33/open_spiel/python/algorithms/losses/rl_losses.py#L94

@fzvinicius can you confirm this?

**Comment by lanctot** [comment]
@fzvinicius can you take a look?

**Comment by lanctot** [comment]
I noticed our Jax policy gradients does something similar where it adds the entropy_loss: https://github.com/deepmind/open_spiel/blob/b03a1977315ffcc1c773fbe1eee7c18f10f96a1e/open_spiel/python/jax/policy_gradient.py#L80

but in that case it gets the loss from rlax, which returns the negative entropy: https://github.com/deepmind/rlax/blob/d6bdb68edc88137bf9ac65e12bd036c888b7dfa0/rlax/_src/policy_gradients.py#L121

**Comment by lanctot** [comment]
This is indeed a bug, thanks for finding. We're fixing internally and should be merged in the next day or two.
